app/src/main/java/app/revanced/manager/ui/viewmodel/AboutViewModel.kt

```diff
@@ -15,7 +16,7 @@ class AboutViewModel(private val reVancedAPI: ReVancedAPI) : ViewModel() {
 
     init {
         viewModelScope.launch {
-            withContext(Dispatchers.IO) { reVancedAPI.getSocials().getOrNull() }?.let(
+            withContext(Dispatchers.IO) { reVancedAPI.getSocials("https://api.revanced.app").getOrNull() }?.let(
```

Comment:
Why are you hardcoding the API?

Reply:
Because this is the social page for the manager itself.

Reply:
A separate about page can show if it detects that the api has change to something other than revanced but this PR is about adding social icons to the ReVanced about page.

Reply:
I don't believe the solution for that is to hardcode the API URL, worse case scenario can't you get the default API key?

Nevertheless, it's rather odd to do it like this, since all the API-dependent screens can also hold the same argument of "it's for the manager itself", updates are for the manager but changing the API URL still changes where updates are received from

Reply:
Ok good point.

Reply:
The idea is that the app is branded regardless of changing the API because the same people still developed the app that the branding is for. Implementation wise, I agree, saving the default API URL somewhere as a constant and reading it here sounds like a good idea.

Reply:
Which file should hold the default?

Reply:
That is just... kind of against the whole reason we added branding to API for

If the plan is to essentially soft-hardcode, just hardcode altogether, there was no reason to change the ReVanced icon to work through the API especially considering that even if we change the logo remotely, that doesn't mean the app logo updates

Not only is loading the image remotely worse, but it can be inconsistent with the current app icon and also make the screen unnecessarily dependent on the internet

Reply:
Yeah I agree

Reply:
> If the plan is to essentially soft-hardcode, just hardcode altogether

In this case, we would not be able to update the links. The point is that the API can ship its own links, but that would be added somewhere else instead of the about page of ReVanced Manager, whereas the about page will have the branding of its origin with the ability to update links if needed.

Reply:
Personally I think that all the links should be hardcoded, the argument that "We would have to update the app to change links" is kind of pointless when we are still updating the app for other reasons quite often.

Reply:
If the API changes then the second about page that loads socials from the new API could be used though.

Reply:
Why will there be a second about page?? what did I miss?

Reply:
In this case, the links from the API can be shown somewhere in the screen for the API.

Reply:
In the event that someone makes their own API, then users should have option to contact them in case something goes wrong.

Reply:
Though adding a second about page is not what this PR is about.

Reply:
| In this case, the links from the API can be shown somewhere in the screen for the API.

Yeah that is a good idea.

Reply:
@oSumAtrIX So do we return to hardcoding the manager social links?

Reply:
Consuming the API is smarter as you can update everything remotely. What is the argument against that?

Reply:
Half a second delay from opening the about page to showing the socials. Plus greater control of how they are laid out if they are hardcoded.

Reply:
> Half a second delay from opening the about page to showing the socials using the API

You can preload and cache requests.

> Plus greater control of how they are laid out if they are hardcoded.

Can you explain?

Reply:
> Can you explain?

For example, with the donation links we used to have a link to https://revanced.app/donate but now there is no link to there on the API so it doesn't show up. I guess you could add it to the socials or donations sections of the API, so I guess I don't really know.

Reply:
Yeah, it can be added to the API in case we want to, though we have a dedicated endpoint for that in case we want to implement it: https://api.revanced.app/v2/donations

Reply:
Yeah that is where I was thinking it might get placed.